### PR TITLE
Operator docs for Source.tick

### PR DESCRIPTION
**Pull request by johanandren**
Refs #25468

### CODE REVIEW

File: akka-docs/src/main/paradox/stream/operators/Source/tick.md [hunk 1]
```diff
@@ -15,15 +15,55 @@ A periodical repetition of an arbitrary object.
 ## Description
 
 A periodical repetition of an arbitrary object. Delay of first tick is specified
-separately from interval of the following ticks.
+separately from interval of the following ticks.
```
Comment (ennru) [inline akka-docs/src/main/paradox/stream/operators/Source/tick.md]:
```
A source periodically emitting an arbitrary element. The delay of the first tick is specified
separately from the interval of the following ticks. 
```

File: akka-docs/src/main/paradox/stream/operators/Source/tick.md [hunk 2]
```diff
@@ -15,15 +15,55 @@ A periodical repetition of an arbitrary object.
...
-separately from interval of the following ticks.
+separately from interval of the following ticks. 
+
+If downstream is applying backpressure when the time period has passed the tick is dropped.
+
+The source materializes a @apidoc[Cancellable] that can be used to complete the source.
+
+@@@note
+
+The element must be immutable as the source can be materialized several times and may pass between threads, see second
```
Comment (ennru) [inline akka-docs/src/main/paradox/stream/operators/Source/tick.md]:
```suggestion
The element must be immutable as the source can be materialized several times and may pass it between threads, see the second 
```

File: akka-docs/src/main/paradox/stream/operators/Source/tick.md [hunk 3]
```diff
@@ -15,15 +15,55 @@ A periodical repetition of an arbitrary object.
...
+@@@note
+
+The element must be immutable as the source can be materialized several times and may pass between threads, see second 
+example for achieving a periodical element that changes over time.
+
+@@@
+
+## Examples
+
+Let's start with a simple sample, just printing to standard out periodically:
```
Comment (ennru) [inline akka-docs/src/main/paradox/stream/operators/Source/tick.md]:
```suggestion
This first example prints to standard out periodically:
```

File: akka-docs/src/main/paradox/stream/operators/Source/tick.md [hunk 4]
```diff
@@ -15,15 +15,55 @@ A periodical repetition of an arbitrary object.
...
+
+The source materializes a @apidoc[Cancellable] that can be used to complete the source.
+
+@@@note
+
+The element must be immutable as the source can be materialized several times and may pass between threads, see second 
+example for achieving a periodical element that changes over time.
+
+@@@
+
```
Comment (ennru) [inline akka-docs/src/main/paradox/stream/operators/Source/tick.md]: Possibly add a reference to `throttle`.

**Review by ennru — APPROVED** [review]
LGTM.

File: akka-docs/src/main/paradox/stream/operators/Source/tick.md [hunk 5]
```diff
@@ -15,15 +15,54 @@ A periodical repetition of an arbitrary object.
...
+separately from interval of the following ticks. 
+
+If downstream is applying backpressure when the time period has passed the tick is dropped.
+
+The source materializes a @apidoc[Cancellable] that can be used to complete the source.
+
+@@@note
+
+The element must be immutable as the source can be materialized several times and may pass it between threads, see the second 
+example for achieving a periodical element that changes over time.
```
Comment (helena) [inline akka-docs/src/main/paradox/stream/operators/Source/tick.md]: A bit of a run on.

File: akka-docs/src/main/paradox/stream/operators/Source/tick.md [hunk 6]
```diff
@@ -15,15 +15,54 @@ A periodical repetition of an arbitrary object.
...
+
+Scala
+:   @@snip [Tick.scala](/akka-docs/src/test/scala/docs/stream/operators/source/Tick.scala) { #simple }
+
+Java
+:   @@snip [Tick.java](/akka-docs/src/test/java/jdocs/stream/operators/source/Tick.java) { #simple }
+
+You can also use the tick to periodically emit a value, in this sample we use the tick to trigger a query to an
+actor using @ref:[ask](../../../typed/interaction-patterns.md#outside-ask) and emit the response downstream. For this
+usage we don't really care about the actual tick value, only that it was emitted:
```
Comment (helena) [inline akka-docs/src/main/paradox/stream/operators/Source/tick.md]:
```suggestion
usage, what is important is that it was emitted, not the actual tick value.
```

**Review by chbatey — APPROVED** [review]

**Pull request merged**